ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -216,6 +216,14 @@
   public static final String storeReadWriteEnableSizeThresholdPercentageDeltaName =
       "store.read.write.enable.size.threshold.percentage.delta";
 
+  /**
+   * Don't do zero copy but pre-fetch data from disk if true.
+   */
+  @Config(storeEnableDataPreFetchName)
+  @Default("true")
```

Comment:
false by default since that is the current behavior

Reply:
fixed.